MediaBrowser.Common/Updates/IInstallationManager.cs

```diff
@@ -42,9 +46,11 @@ public interface IInstallationManager : IDisposable
         /// <returns>All plugins matching the requirements.</returns>
         IEnumerable<PackageInfo> FilterPackages(
             IEnumerable<PackageInfo> availablePackages,
-            string name = null,
-            Guid guid = default,
-            Version specificVersion = null);
+            string? name = null,
+#pragma warning disable CA1720 // Identifier contains type name
+            Guid? guid = default,
```

Comment:
can rename this also

Reply:
Now this is too much of a co-incidence - something is definitely going amiss with VS's search function.

Reply:
done.